vips/vip-256/bscmainnet.ts

```diff
@@ -0,0 +1,343 @@
+import { BigNumber } from "ethers";
+import { parseUnits } from "ethers/lib/utils";
+
+import { ProposalType } from "../../src/types";
+import { makeProposal } from "../../src/utils";
+
+export const TREASURY = "0xF322942f644A996A617BD29c16bd7d231d9F35E9";
+export const TOKEN_REDEEMER = "0x29171F17BF7F3691908eD55bAC2014A632B87dD3";
+export const vBNB = "0xA07c5b74C9B40447a954e1466938b865b6BBea36";
+export const NORMAL_TIMELOCK = "0x939bD8d64c0A9583A7Dcea9933f7b21697ab6396";
+export const WBNB = "0xbb4CdB9CBd36B01bD1cBaEBF2De08d9173bc095c";
+export const BINANCE = "0x6657911F7411765979Da0794840D671Be55bA273";
+
+export const WBNB_AMOUNT = parseUnits("5901.651898613689829191", 18).toString();
+export const vBNB_AMOUNT = parseUnits("672665.16913031", 8).toString();
+export const BNB_AMOUNT_TO_BINANCE = BigNumber.from(parseUnits("15636.74", 18))
+  .add(parseUnits("5865.60", 18))
+  .toString();
+
+// Received from vBNB after redeem is 15847.979107135037434728. But we will send to treasury only 15847.679107135037434728 i.e., 0.3 is kept in the timelock
+export const BNB_AMOUNT_TO_TREASURY = parseUnits("15847.679107135037434728", 18).toString();
+
+interface VTokenTransfer {
+  symbol: string;
+  amount: string;
+  address: string;
+}
+
+interface BEP20Transfer {
+  symbol: string;
+  amount: string;
+  address: string;
+}
+
+export const vToken_Transfers = [
+  {
+    symbol: "vBTC",
+    amount: parseUnits("344.60553433", 8).toString(),
+    address: "0x882C173bC7Ff3b7786CA16dfeD3DFFfb9Ee7847B",
+  },
+  {
+    symbol: "vETH",
+    amount: parseUnits("1685.06190157", 8).toString(),
+    address: "0xf508fCD89b8bd15579dc79A6827cB4686A3592c8",
+  },
+  {
+    symbol: "vUSDT",
+    amount: parseUnits("2554859.57915445", 8).toString(),
+    address: "0xfD5840Cd36d94D7229439859C0112a4185BC0255",
+  },
+  {
+    symbol: "vLTC",
+    amount: parseUnits("27660.07068667", 8).toString(),
+    address: "0x57A5297F2cB2c0AaC9D554660acd6D385Ab50c6B",
+  },
+  {
+    symbol: "vXVS",
+    amount: parseUnits("138740.90847753", 8).toString(),
+    address: "0x151B1e2635A717bcDc836ECd6FbB62B674FE3E1D",
+  },
+  {
+    symbol: "vCAKE",
+    amount: parseUnits("245187.19242229", 8).toString(),
+    address: "0x86aC3974e2BD0d60825230fa6F355fF11409df5c",
+  },
+  {
+    symbol: "vUNI",
+    amount: parseUnits("2497.74135186", 8).toString(),
+    address: "0x27FF564707786720C71A2e5c1490A63266683612",
+  },
+  {
+    symbol: "vFDUSD",
+    amount: parseUnits("10355.09068368", 8).toString(),
+    address: "0xC4eF4229FEc74Ccfe17B2bdeF7715fAC740BA0ba",
+  },
```

Comment:
Let's review if we should try to redeem vFDUSD. The launchpool will decrease the liquidity in the market